package.json

```diff
@@ -35,7 +35,6 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.2",
-    "@babel/plugin-proposal-class-properties": "^7.7.0",
```

Comment:
You removed it here but it's still present in the babelrc file, is it expected?
https://github.com/heetch/flamingo/pull/132/commits/c78d737bad16f82a85815504b059ffb08725c508#diff-e56633f72ecc521128b3db6586074d2cR4

Reply:
good catch 😅